x-pack/plugins/security/server/authorization/authorization_service.ts

```diff
@@ -158,12 +151,23 @@ export class AuthorizationService {
     return authz;
   }
 
-  start({ clusterClient, features }: AuthorizationServiceStartParams) {
+  start({ clusterClient, features, online$ }: AuthorizationServiceStartParams) {
```

Comment:
__note:__ we need similar code for the Session Index - decided to generalize. Hopefully we can ditch our workaround as soon as Core provides us with an alternative.

Reply:
Yeah, not exactly the same, but they are closely related. Will mention https://github.com/elastic/kibana/issues/41983, thanks!